feature/aft/aft_base/otg_tests/afts_base/README.md

```diff
@@ -77,7 +71,7 @@ Bring down both links between ATE:port1,port2 and DUT:port1,port2 using OTG api.
 
 *   BGP routes advertised from ATE:port1,port2 must be removed from RIB,FIB of the DUT, query results nil.
 *   IS-IS routes advertised from ATE:port3 must have one next hop.
-*   BGP routes advertised from ATE:port3 must have 2 next hops pointing to tunnels.
+*   BGP routes advertised from ATE:port3 must have indirect next hop.
 *   For verifying prefix, nexthop groups, next hop use the leaves mentioed in the path section.
 *   Verify afts prefix advertised by BGP,ISIS.
 *   Verify its next hop group, number of next hop and its interfaces.
```

Comment:
Same comment here about defining what verification we are talking about.

Reply:
will update